src/CommunityToolkit.Maui.Core/Layouts/UniformItemsLayoutManager.shared.cs

```diff
@@ -86,7 +86,7 @@ public override Size Measure(double widthConstraint, double heightConstraint)
 	int GetColumnsCount(int visibleChildrenCount, double widthConstraint)
 	{
 		var columnsCount = visibleChildrenCount;
-		if (childWidth != 0 && !double.IsPositiveInfinity(widthConstraint))
+		if (visibleChildrenCount > 0 && childWidth != 0 && !double.IsPositiveInfinity(widthConstraint))
```

Comment:
It doesn't seem to be correct. We should have at least one column, but in your case we may get 0.